package Exception.src;
public class Case1Exception {
    public static void main(String[] args) {
      try {
          System.out.println("Mohammad");
      }
      catch (ArithmeticException b){
          System.out.println("Huzaifa");
      }
      finally {
          System.out.println("Subhanallha");
      }
    }
}
